src/django/api/reports/pending_match_records_for_certain_api_users.sql

```diff
@@ -0,0 +1,17 @@
+SELECT 
+    src.contributor_id,
+    ac.name AS contributor_name,
+    COUNT(*) AS pending_matches_count
+FROM 
+    api_facilitymatch afm
+JOIN 
+    api_facilitylistitem afli ON afm.facility_list_item_id = afli.id
+JOIN 
+    api_source src ON afli.source_id = src.id
+JOIN
+    api_contributor ac ON src.contributor_id = ac.id
+WHERE 
+    afm.status = 'PENDING'
+    AND src.contributor_id IN (2060, 1045, 685, 3356)
```

Comment:
Could you explain why do we focus on these contributors - 2060, 1045, 685, 3356?

Reply:
According to the goal of the current task, this report should be generated for particular users: [Textile Exchange](https://opensupplyhub.org/admin/api/contributor/2060/change/), [ZDHC](https://opensupplyhub.org/admin/api/contributor/1045/change/), [Worldly](https://opensupplyhub.org/admin/api/contributor/685/change/), [Amfori](https://opensupplyhub.org/admin/api/contributor/3356/change/)